Description: Typo fix for plugin if_. It has been fixed upstream in revision
 2224, but is not present in tags/1.2.6 or tags/1.3.4
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=523765
Origin: other, http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=523765
Last-Update: 2009-07-19
Index: munin-1.2.6/node/node.d.freebsd/if_.in
===================================================================
--- munin-1.2.6.orig/node/node.d.freebsd/if_.in	2009-05-28 20:06:12.000000000 +0200
+++ munin-1.2.6/node/node.d.freebsd/if_.in	2009-05-28 20:06:23.000000000 +0200
@@ -47,7 +47,7 @@
 	echo 'graph_args --base 1000'
 	echo 'graph_vlabel bits per ${graph_period} in (-) / out (+)'
 	echo 'graph_category network'
-	echo "graph_info This graph shows the traffic of the $INTERFACE network interface. Please note that the traffic is shown in bits per second, not bytes. IMPORTANT: Since the data source for this plugin use 32bit counters, this plugin is really unreliable and unsuitable for most 100Mb (or faster) interfaces, where bursts are expected to exceed 50Mbps. This means that this plugin is usuitable for most production environments."
+	echo "graph_info This graph shows the traffic of the $INTERFACE network interface. Please note that the traffic is shown in bits per second, not bytes. IMPORTANT: Since the data source for this plugin use 32bit counters, this plugin is really unreliable and unsuitable for most 100Mb (or faster) interfaces, where bursts are expected to exceed 50Mbps. This means that this plugin is unsuitable for most production environments."
 	echo 'rbytes.label received'
         echo 'rbytes.type COUNTER'
         echo 'rbytes.graph no'
Index: munin-1.2.6/node/node.d.linux/if_.in
===================================================================
--- munin-1.2.6.orig/node/node.d.linux/if_.in	2009-05-28 20:06:12.000000000 +0200
+++ munin-1.2.6/node/node.d.linux/if_.in	2009-05-28 20:06:26.000000000 +0200
@@ -107,7 +107,7 @@
 	echo 'graph_args --base 1000'
 	echo 'graph_vlabel bits in (-) / out (+) per ${graph_period}'
 	echo 'graph_category network'
-	echo "graph_info This graph shows the traffic of the $INTERFACE network interface. Please note that the traffic is shown in bits per second, not bytes. IMPORTANT: Since the data source for this plugin use 32bit counters, this plugin is really unreliable and unsuitable for most 100Mb (or faster) interfaces, where bursts are expected to exceed 50Mbps. This means that this plugin is usuitable for most production environments. To avoid this problem, use the ip_ plugin instead."
+	echo "graph_info This graph shows the traffic of the $INTERFACE network interface. Please note that the traffic is shown in bits per second, not bytes. IMPORTANT: Since the data source for this plugin use 32bit counters, this plugin is really unreliable and unsuitable for most 100Mb (or faster) interfaces, where bursts are expected to exceed 50Mbps. This means that this plugin is unsuitable for most production environments. To avoid this problem, use the ip_ plugin instead."
 	echo 'down.label received'
         echo 'down.type COUNTER'
         echo 'down.graph no'
